src/Tribe/Commerce/PayPal/Orders/Tabbed_View.php

```diff
@@ -32,8 +32,8 @@ public function filter_tribe_tickets_orders_tabbed_view_tab_map( array $tab_map
 	 */
 	public function register_orders_tab( Tribe__Tabbed_View $tabbed_view, WP_Post $post ) {
 		$provider = Tribe__Tickets__Tickets::get_event_ticket_provider( $post->ID );
-		$not_using_paypal = 'Tribe__Tickets__Commerce__PayPal__Main' !== $provider;
-		$has_no_paypal_tickets = count( Tribe__Tickets__Commerce__PayPal__Main::get_instance()->get_tickets_ids( $post ) ) === 0;
+		$not_using_paypal = ! $provider instanceof Tribe__Tickets__Commerce__PayPal__Main;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space

<sub>hash: a7803505c2cc9300db747ccdc08c8101</sub>

Reply:
fixed